gradle/libs.versions.toml

```diff
@@ -68,7 +68,7 @@ protobuf = "3.22.3"
 paparazzi = "1.0.0"
 paparazziNative = "2022.1.1-canary-f5f9f71"
 shadow = "8.1.1"
-skiko = "0.8.17"
```

Comment:
We need to fix tests, adding more relaxed checks for values

Reply:
Still has failed tests

Reply:
accidentally made tolerance too small, now should be good